principles/coding-principles.md

```diff
@@ -63,24 +63,32 @@ The larger the chunk of work you’re estimating, the more inaccurate you will b
 
 ### 1. Follow the Microsoft coding standards
 The Microsoft developed [Coding Conventions](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/inside-a-program/coding-conventions) were  developed to give a consistent look and feel to C# code so that readers can focus on the content and not the layout. This can help reduce the congitive over-head and enable readers to understand the code more quickly based on their previous experience.
-  
-### 2. Use static code analysis
-To ensure that code meets the quality criteria of the Apprenticeship Service static code analysis via SonarCloud.io should be utilised and integrated into the build pipelines. The output of this analysis should be reviewed as part of the Code Review process and any issues rectified
 
-## Testing-Specific Principles
+### 2. SOLID
+Follow [SOLID principles](https://en.wikipedia.org/wiki/SOLID). This will make code more understandable and easier to maintain
+* Single Responsibility - 
+* Open/Closed principle
+* Liskov Substitution
+* Interface Segregation
+* Dependency Inversion
 
-### 1. Write tests
+### 3. Use static code analysis
+To ensure that code meets the quality criteria of the Apprenticeship Service static code analysis via SonarCloud.io should be utilised and integrated into the build pipelines. The output of this analysis should be reviewed as part of the Code Review process and any issues highlighted should be rectified
```

Comment:
Could you include a link to our SonarCloud instance to help people understand where it is as they are reading through the guidance.

Reply:
I've added a link in

Reply:
I was thinking of this link https://sonarcloud.io/organizations/educationandskillsfundingagency/projects

Reply:
Its not necessary to provide the full link as when you log in with an SFA GitHub account you automatically get taken to that page